CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * [ENHANCEMENT] Distributor: Include `X-Scope-OrgId` header in requests forwarded to configured forwarding endpoint. #3283
 * [ENHANCEMENT] Alertmanager: reduced memory utilization in Mimir clusters with a large number of tenants. #3309
 * [ENHANCEMENT] Add experimental flag `-shutdown-delay` to allow components to wait after receiving SIGTERM and before stopping. In this time the component returns 503 from /ready endpoint. #3298
+* [ENHANCEMENT] Align the `-server.http-write-timeout` with `-querier.timeout` to 2m. #3346
```

Comment:
We use CHANGE for any change to default config. Please move it above among other CHANGE entries.

```suggestion
* [CHANGE] The default value of `-server.http-write-timeout` has changed from 30s to 2m. #3346
```